src/models/log.ts

```diff
@@ -291,12 +292,30 @@ class LoggerWrapper {
         };
     }
 
+    debugLogging(debug = false, ignoreUrls = ['/status']) {
+        return async (ctx: any, next: Function) => {
```

Comment:
why the Any? (also I see I missed it in the basicLogMiddleware too
install @types/koa and use Koa.Context instead

Reply:
fixed